assets/js/WfModuleContextMenu.js

```diff
@@ -133,7 +131,7 @@ export default class WfModuleContextMenu extends React.Component {
 
     return (
       <Modal isOpen={this.state.exportModalOpen} toggle={this.toggleExportModal} className={this.props.className}>
-        <ModalHeader toggle={this.toggleModal} className='dialog-header modal-header d-flex align-items-center' >
+        <ModalHeader className='dialog-header modal-header d-flex align-items-center' >
```

Comment:
Why remove toggleModal? What bug is this?

Reply:
We have our own custom close icon in the modal:

<div className='icon-close' onClick={this.toggleExportModal}></div>

We could instead use `toggle={this.toggleModal}` in the ModalHeader, but we would be using the default 'X' icon to close, instead of our custom one.